### ISSUE TITLE
Can't get more than 57% accuracy on different models

### ISSUE THREAD

**Issue by VEXstar** [qubvel/segmentation_models.pytorch#391]
Hello everyone, I'm new to machine learning, and as a start I decided to try this library.
The problem is that I tried several models, encoders, loss functions, but none of the combinations gave more than 57% accuracy in validation. However, on the training sample, the accuracy reached 97%.
As a dataset, I use multiple chest CT scans (2k slices in total) with matching masks.
Masks have a binary format (0-background, 1-mask)
Images are stored in a grayscale.
I'm sure the problem is not related to this library, but I hope for help.

**Comment by AlphonsG** [comment]
Hello @VEXstar, did you resolve this?
